es6-const-let/index.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>es6-const-let</title>
+</head>
+<body>
+  <script>
+    const greetings = [
+      'hello!',
+      'konichiwa!',
+      'nihao',
+      'bonjour',
+      'gutten tag',
+      'ola'
+    ];
+
+    function logAllGreetings(greetingList) {
+      let i = 0;
+      while (i < greetingList.length) {
+        console.log('Current greeting:', greetingList[i]);
+        i++;
+      }
+    }
+    logAllGreetings(greetings);
+
+    function getRandomGreeting(greetingList) {
+      let randomIndex = Math.floor(Math.random() * greetingList.length);
```

Comment:
This does not get re-assigned within a single run of this scope, so it should be a constant.

---

![Screenshot_2](https://user-images.githubusercontent.com/16143771/115123002-da71a800-9f6f-11eb-9e36-23c155aa71b0.jpg)